scripts/ci/test_reborn_pr_test_plan.py

```
@@ -1025,6 +1025,98 @@ def test_shared_sccache_action_widens_to_exhaustive_plan(self) -> None:
+                    {"name": "selected", "packages": ["alpha", "beta", "gamma"]}
+                ],
+                "root_partitions": [0, 1, 2, 3],
+                "integration_lanes": [0, 1, 2, 3, "groups"],
+                "run_group_tests": True,
+                "run_qa_replay": True,
+                "run_sandbox_docker": True,
+                "coverage_mode": "full",
+            },
+        )
+
+        with self.assertRaisesRegex(ValueError, "unmapped test or CI path"):
```

Ruff (0.16.2)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)